src/renderer/components/+events/events.tsx

```diff
@@ -37,23 +40,60 @@ const defaultProps: Partial<Props> = {
 export class Events extends React.Component<Props> {
   static defaultProps = defaultProps as object;
 
-  get store() {
+  private sortingCallbacks: TableSortCallbacks = {
+    [columnId.namespace]: (event: KubeEvent) => event.getNs(),
+    [columnId.type]: (event: KubeEvent) => event.type,
+    [columnId.object]: (event: KubeEvent) => event.involvedObject.name,
+    [columnId.count]: (event: KubeEvent) => event.count,
+    [columnId.age]: (event: KubeEvent) => event.getTimeDiffFromNow(),
+  };
```

Comment:
We are supposed to be only fixing stuff. Please back out all refactors.

Reply:
All these relevant to actual fix and `compact=true` from previous PR(s).

Reply:
It seems like a lot of work, why not just correctly handle slicing in `KubeObjectListLayout`? Namely sorting and then only displaying a slice?

Reply:
Because I need access to all items at `Events` page (e.g. for customizing header)